chunker/application.py

```diff
@@ -0,0 +1,61 @@
+from util import log
+
+import pipeline, tiler
+import re, os, sys, time
+import boto.sqs
+
+def parse_filename(path):
```

Comment:
Since `application.py` is heavily dependent on architecture which is subject to change I haven't touched this file yet.